dags/main_summary.py

```diff
@@ -336,45 +336,12 @@
     executor=GetDefaultExecutor(),
     dag=dag)
 
-clients_daily_v6 = EMRSparkOperator(
-    task_id="clients_daily_v6",
-    job_name="Clients Daily v6",
-    owner="relud@mozilla.com",
-    email=["telemetry-alerts@mozilla.com", "relud@mozilla.com"],
-    execution_timeout=timedelta(hours=5),
-    instance_count=10,
-    env=tbv_envvar("com.mozilla.telemetry.views.ClientsDailyView", {
-        "date": "{{ ds_nodash }}",
-        "input-bucket": "{{ task.__class__.private_output_bucket }}",
-        "output-bucket": "{{ task.__class__.private_output_bucket }}"
-    }),
-    uri="https://raw.githubusercontent.com/mozilla/telemetry-airflow/master/jobs/telemetry_batch_view.py",
-    dag=dag)
-
-register_status(clients_daily_v6, "Clients Daily", "A view of main pings with one row per client per day.")
-
-clients_daily_v6_bigquery_load = SubDagOperator(
-    subdag=load_to_bigquery(
-        parent_dag_name=dag.dag_id,
-        dag_name="clients_daily_v6_bigquery_load",
-        default_args=default_args,
-        dataset_s3_bucket="telemetry-parquet",
-        aws_conn_id="aws_dev_iam_s3",
-        dataset="clients_daily",
-        dataset_version="v6",
-        gke_cluster_name="bq-load-gke-1",
-        reprocess=True,
-        bigquery_dataset="telemetry_derived",
-        cluster_by=["sample_id"],
-        rename={"submission_date_s3": "submission_date"},
-        replace=["SAFE_CAST(sample_id AS INT64) AS sample_id"],
-        ),
-    task_id="clients_daily_v6_bigquery_load",
-    dag=dag)
+register_status(clients_daily, "Clients Daily", "A view of main pings with one row per client per day.")
 
 clients_last_seen = bigquery_etl_query(
     task_id="clients_last_seen",
     destination_table="clients_last_seen_v1",
+    project_id="moz-fx-data-shared-prod",
```

Comment:
when this is merged the most recent day of `clients_last_seen_v1` will need to be copied from `moz-fx-data-derived-datasets` to `moz-fx-data-shared-prod` so that it can referenced in the next run.